mmv1/third_party/terraform/website/docs/r/sql_user.html.markdown

```diff
@@ -155,6 +157,15 @@ The read only `password_policy.status` subblock supports:
 
 * `password_expiration_time` - (read only) Password expiration duration with one week grace period.
 
+## Ephemeral Attributes Reference
+
+The following write-only attributes are supported:
+
+* `password_wo` -
```

Comment:
`password` has considerably more documentation. Should more be added here, to make it more similar?

Reply:
updated to match `password` doc